examples/vuejs/with-thirdpartyemailpassword/src/views/AuthView.vue

```diff
@@ -0,0 +1,593 @@
+<script lang="ts">
+import ThirdPartyEmailPassword from "supertokens-web-js/recipe/thirdpartyemailpassword";
+
+const websitePort = import.meta.env.VUE_APP_WEB_PORT || 3000;
+const websiteDomain = import.meta.env.VUE_APP_WEB_URL || `http://localhost:${websitePort}`;
+
+export default {
+    data() {
+        return {
+            isSignIn: true,
+            email: "",
+            password: "",
+            error: false,
+            errorMessage: "Something went wrong",
+        };
+    },
+
+    mounted() {
+        const params = new URLSearchParams(window.location.search);
+
+        if (params.has("error")) {
+            this.errorMessage = "Something went wrong";
+            this.error = true;
+        }
+    },
+
+    methods: {
+        goToSignUp() {
+            this.isSignIn = false;
+        },
+        goToSignIn() {
+            this.isSignIn = true;
+        },
+        signIn: async function (_: SubmitEvent) {
+            if (
+                !(
+                    await ThirdPartyEmailPassword.doesEmailExist({
+                        email: this.email,
+                    })
+                ).doesExist
+            ) {
+                this.errorMessage = "Email does not exist, sign up instead";
+                this.error = true;
+                return;
+            }
```

Comment:
I don't think you need this check.. usually in sign in page, an email that doesn't exist yieds a wrong credential error.

Reply:
This was a replacement for how we check if an email exists after the user enters it during sign up, but I can remove it and rely on the signin/up API instead